packages/vaadin-icon/src/vaadin-iconset.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+import { nothing, svg } from 'lit';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+
+const iconRegistry = {};
+
+export const getIconId = (id) => (id || '').replace('vaadin-icon:', '');
+
+/**
+ * `<vaadin-iconset>` is a Web Component for creating SVG icon collections.
+ *
+ * @extends HTMLElement
+ * @mixes ElementMixin
+ */
+class IconsetElement extends ElementMixin(PolymerElement) {
+  static get template() {
+    return null;
+  }
+
+  static get is() {
+    return 'vaadin-iconset';
+  }
+
+  static get version() {
+    return '21.0.0-alpha2';
+  }
+
+  static get properties() {
+    return {
+      /**
+       * The name of the iconset.
+       */
+      name: {
+        type: String,
+        observer: '__nameChanged'
+      }
+    };
+  }
+
+  /**
+   * Create an instance of the iconset.
+   *
+   * @param {string} name
+   */
+  static getIconset(name) {
+    let iconset = iconRegistry[name];
+    if (!iconset) {
+      iconset = document.createElement('vaadin-iconset');
+      iconset.name = name;
+      iconRegistry[name] = iconset;
+    }
+    return iconset;
+  }
+
+  /** @protected */
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.style.display = 'none';
+  }
+
+  /**
+   * Produce SVGTemplateResult for the element matching `id` in this
+   * iconset, or `undefined` if there is no matching element.
+   *
+   * @param {string} id
+   */
+  applyIcon(id) {
+    // create the icon map on-demand, since the iconset itself has no discrete
+    // signal to know when it's children are fully parsed
+    this._icons = this._icons || this.__createIconMap();
+    return this.__prepareSvg(this._icons[getIconId(id)]);
+  }
+
+  /**
+   * Create a map of child SVG elements by id.
+   */
+  __createIconMap() {
+    const icons = Object.create(null);
```

Comment:
I looked across the PR, but didn't find the reason `Object.create` is used instead of `{}`. 

Could you explain this line?

Reply:
This line comes from [iron-iconset-svg](https://github.com/PolymerElements/iron-iconset-svg/blob/c575f7f30be3174607a76ccf15c16ce4c2416f4c/iron-iconset-svg.js#L203-L206) but that's really old code and the issue mentioned there isn't present in Firefox anymore.

Reply:
Updated to use `{}`.